app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
Great job team 🤝
- As stated in the project requirements, kindly [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.
- Your design
![login](https://github.com/Abdulwhbhussain/blog-app/assets/63917725/cfa8b8f2-a93c-4b77-bb1a-618d4ad90223)
- Given design
![given-login](https://github.com/Abdulwhbhussain/blog-app/assets/63917725/653db402-4d20-4e70-a327-1b7073e30799)

Reply:
Did it Bro!